contracts/optimizedForceMove.sol

```diff
@@ -162,12 +162,115 @@ contract OptimizedForceMove {
             keccak256(abi.encode(variableParts[variableParts.length - 1].outcome))
         );
 
-        emit ForceMove(channelId, now + fixedPart.challengeDuration); // TODO what else should go in here?
+        emit ForceMove(channelId, now + fixedPart.challengeDuration, largestTurnNum, challenger); // TODO what else should go in here?
 
         channelStorageHashes[channelId] = keccak256(abi.encode(channelStorage));
 
     }
 
+    function respond(
+        uint256 turnNumRecord,
+        uint256 finalizesAt,
+        address challenger,
+        bool[2] memory isFinalAB,
```

Comment:
It might be simpler to stick with `isFinalCount` or whatever it was across all methods. It's marginal though.

Reply:
I did consider doing that, but as you say it is marginal. This way mirrors the `variablePartAB` convention, so at least is consistent with something...

It is also the minimal amount of information. Using a `uint` would be overkill (we only ever consider two commitments in this method, and in `_validTransition`) and require slightly more computation to transcode the information into the `State` structs.